### DISCUSSION TITLE
Implementing the Cyclone II ep2c5t144 FPGA (And other popular FPGAs)

### DISCUSSION THREAD

**Discussion by Slicudis** [logisim-evolution/logisim-evolution#2008] [Ideas]
I've got an idea: Add the Cyclone II ep2c5t144
Reason: It's one of the most popular and used FPGAs, specially for people who can't afford big FPGAs
![image](https://github.com/logisim-evolution/logisim-evolution/assets/140275002/373183fd-7c32-40ce-81df-c024052034cf)

**Comment by maehne** [comment]
Cyclone II FPGAs are only supported up to Quartus II 13.1, which [Web Edition](https://www.intel.com/content/www/us/en/collections/products/fpga/software/downloads.html?edition=web&platform=windows&f:guidetmD240C377263B4C70A4EA0E452D0182CA=%5BQuartus%C2%AE%20II%20Software%5D) is still available for download. If you have succeeded to install it, you should be able to use it for synthesis in Logisim-evolution by setting the Quartus toolpath in Window -> Preferences -> Software. You can then add support for the board yourself by following [these rough instructions](https://github.com/logisim-evolution/logisim-evolution/issues/376). We also welcome PRs adding support for new development boards.

I am closing this as resolved. Feel free to reopen if you have further questions regarding this topic.
